src/tournamentcreator.cpp

```diff
@@ -56,12 +56,15 @@ void MainWindow::on_ContinueTournamentCreatorButton_clicked()
     UtilDebug("Tournament Creator Continue Button Clicked");
     if(m_TempPlayerList->size() > 0)
     {
-        m_MainPlayerList.clear();
+        m_MainPlayerList->clear();
         for(const auto &player : *m_TempPlayerList)
         {
-            m_MainPlayerList.push_back(player);
+            UtilLog(player.GetName());
+            m_MainPlayerList->push_back(player);
         }
 
+        //*m_MainPlayerList = std::move(*m_TempPlayerList);
```

Comment:
Check to see if this now works with the changes added. If not remove the comment.

Reply:
Tested and this now works.